android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/UnityAndroidPayFragment.java

```diff
@@ -0,0 +1,172 @@
+package com.shopify.unity.buy;
+
+import android.app.Fragment;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.wallet.MaskedWallet;
+import com.google.android.gms.wallet.Wallet;
+import com.google.android.gms.wallet.WalletConstants;
+import com.shopify.buy3.pay.PayCart;
+import com.shopify.buy3.pay.PayHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UnityAndroidPayFragment extends Fragment implements GoogleApiClient.ConnectionCallbacks {
+    private PayCart cart;
+    private String unityDelegateObjectName;
+    private String countryCode;
+    private String androidPublicKey;
+    private int androidPayEnvironment;
+    private GoogleApiClient googleApiClient;
+
+    private MaskedWallet maskedWallet;
+
+    private static final String EXTRA_UNITY_DELEGATE_OBJECT_NAME = "unityDelegateObjectName";
+    private static final String EXTRA_PAY_CART = "payCart";
+    private static final String EXTRA_COUNTRY_CODE = "countryCode";
+    private static final String EXTRA_ANDROID_PAY_ENVIRONMENT = "androidPayEnvironment";
+    private static final String EXTRA_PUBLIC_KEY = "publicKey";
```

Comment:
Not sure how this passed the checkstyle, but the order usually first consts and then variables